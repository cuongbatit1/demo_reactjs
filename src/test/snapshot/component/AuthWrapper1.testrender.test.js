import { render, screen } from "@testing-library/react";
import AuthWrapper1 from "../../../component/AuthWrapper1";

test("test renders AuthWrapper1 snapshot", () => {
  const { asFragment } = render(<AuthWrapper1 />);
  expect(asFragment()).toMatchSnapshot();
});
