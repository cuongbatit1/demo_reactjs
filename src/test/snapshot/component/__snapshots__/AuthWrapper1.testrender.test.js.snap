// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test renders AuthWrapper1 snapshot 1`] = `
<DocumentFragment>
  <div
    class="css-1jhze1m"
  />
</DocumentFragment>
`;
